src/selectors/balances.js

```diff
@@ -21,39 +21,54 @@ import { createSelector } from 'reselect';
 import isEmpty from 'lodash.isempty';
 
 // constants
-import { PLR, USD } from 'constants/assetsConstants';
-import { LIQUIDITY_POOLS } from 'constants/liquidityPoolsConstants';
+import { PLR } from 'constants/assetsConstants';
 
 // utils
 import { getTotalBalanceInFiat } from 'utils/assets';
-import { getStreamBalance } from 'utils/sablier';
-import { BigNumber, formatUnits } from 'utils/common';
-import { getPoolStats } from 'utils/liquidityPools';
+import { BigNumber } from 'utils/common';
+import { sum } from 'utils/bigNumber';
+import { isEtherspotAccount } from 'utils/accounts';
 
 // types
 import type { RootReducerState } from 'reducers/rootReducer';
-import type { Rates, Balances, MixedBalance, MixedBalances } from 'models/Asset';
-import type { LendingReducerState } from 'reducers/lendingReducer';
-import type { PoolPrizeInfo } from 'models/PoolTogether';
-import type { SablierReducerState } from 'reducers/sablierReducer';
-import type { RariReducerState } from 'reducers/rariReducer';
-import type { LiquidityPoolsReducerState } from 'reducers/liquidityPoolsReducer';
+import type { Rates, Balances } from 'models/Asset';
+import type { Account } from 'models/Account';
+import type { AccountsTotalBalances, CategoryBalancesPerChain, ChainBalances } from 'models/Home';
 
 // selectors
 import {
   balancesSelector,
   fiatCurrencySelector,
   ratesSelector,
   activeAccountIdSelector,
-  lendingSelector,
-  poolTogetherStatsSelector,
-  sablierSelector,
-  rariSelector,
-  liquidityPoolsSelector,
+  activeAccountSelector,
 } from './selectors';
 import { availableStakeSelector } from './paymentNetwork';
 
 
+const sumBalancesPerChainChain = (
```

Comment:
It seems you're not summing anything here.
```suggestion
const getChainBalancesForCategory = (
```

Reply:
👍